install_config/router/default_haproxy_router.adoc

```diff
@@ -166,6 +169,66 @@ $ oc adm router --dry-run -o yaml --service-account=router
 ----
 endif::[]
 
+[[deploy-router-access-logs]]
+Configuring the Router to Forward HAProxy Logs::
+
+You can configure the router to forward logs generated by HAProxy 
+to an rsyslog sidecar container. The `--extended-logging=true` parameter 
+appends the syslog container to forward HAProxy logs to standard output. 
+
+----
+$ oc adm router --extended-logging=true
+----
+
+For example, the following is the configuration for a router 
+created using `--extended-logging=true`: 
+----
+$ oc get pod router-1-xhdb9 -o yaml
+apiVersion: v1
+kind: Pod
+spec:
+  containers:
+  - env:
+
+    ....
+
+    - name: ROUTER_SYSLOG_ADDRESS <1>
+      value: /var/lib/rsyslog/rsyslog.sock
+
+    ....
+
+ - command: <2>
+   - /sbin/rsyslogd
+   - -n
+   - -i
+   - /tmp/rsyslog.pid
+   - -f
+   - /etc/rsyslog/rsyslog.conf
+   image: registry.redhat.io/openshift3/ose-haproxy-router:v3.11.188
+   imagePullPolicy: IfNotPresent
+   name: syslog
+----
+<1> The parameter creates a socket file for the logs.
+<2> The parameter adds a new container to the router. In the container, 
```

Comment:
~new~

Reply:
Changed to _The `--extended-logging=true` parameter..._ to highlight that the --extended-logging=true` added the socket file and container.